### PR TITLE
Add a new API to fetch stat var group information

### PR DESCRIPTION
**Pull request by shifucun**
Given a stat var group id, this API fetches its child stat var, child stat var groups and parent stat var groups.

This is meant to be used for stat var hierarchy widget that supports incremental data fetch.

### CODE REVIEW

File: internal/server/statvar_group.go [hunk 1]
```diff
@@ -138,3 +141,104 @@ func (s *Server) GetStatVarGroup(
...
+			return nil, err
+		}
+		for _, c := range result.ChildStatVars {
+			c.DisplayName = statVarNames[c.Id]
+		}
+	}
+	return result, nil
+}
+
+func fetchStatVarNames(ctx context.Context, s *Server, statVars []string) (
```
Comment (pradh) [inline internal/server/statvar_group.go:210]: Should we compute the names and put it in the "name" property of StatVar?

File: internal/server/statvar_group.go [hunk 2]
```diff
@@ -138,3 +141,104 @@ func (s *Server) GetStatVarGroup(
...
+		pv := map[string]string{}
+		cpv := []string{}
+		for _, t := range data.Triples {
+			if t.Predicate == "measuredProperty" {
+				name = t.ObjectName + " of"
+			} else if t.Predicate == "constraintProperties" {
+				cpv = append(cpv, t.ObjectID)
+			} else {
+				pv[t.Predicate] = t.ObjectID
+			}
```
Comment (pradh) [inline internal/server/statvar_group.go:230]: There's more special-casing in current logic http://google3/datacommons/prophet/flume_generator/stat_var_group_util.cc?l=320&rcl=377621180. Including that this doesn't ignore dependent PVs.

**Review by pradh — APPROVED** [review]

File: internal/server/statvar_group.go (hunk 1 above)
Comment (shifucun) [reply internal/server/statvar_group.go:210]: That would be nice!

File: internal/server/statvar_group.go (hunk 2 above)
Comment (shifucun) [reply internal/server/statvar_group.go:230]: I will leave it for now and use the name property once that's added in Flume

**Pull request merged**